docs/stacks-guide-aws.md

```diff
@@ -573,7 +573,7 @@ Below we inspect each of these resource classes in more details:
 
   ```yaml
```

Comment:
https://github.com/crossplaneio/crossplane/blob/master/cluster/examples/workloads/kubernetes/wordpress/aws/resource-classes/rds-class.yaml#L14
dbsubnetGroupNameRef -> dbSubnetGroupNameRef

Reply:
How would you feel about opening another PR to address this? After some back and forth I decided to try keep this PR scoped to just bumping the version number. I think this issue needs fixing, but is not caused by or related to this PR.

Reply:
sounds good, created https://github.com/crossplaneio/crossplane/pull/994/